utils/index.js

```diff
@@ -28,10 +28,18 @@ function getScript (name) {
   return path.resolve(path.join(__dirname, `scripts/${name}.sh`))
 }
 
+function JSONreplacer (key, value) {
+  if (typeof value === 'object' && value !== null && value.type === 'Buffer') {
+    return value.data
+  }
+  return value
+}
```

Comment:
Let's have a better name for it. I was really puzzled reading this function name, unused `key` argument, and the cryptic logic.

Reply:
Let me add a comment there.

Reply:
Added.